api/appsettings.go

```diff
@@ -16,11 +16,17 @@ type AppSettings struct {
 	// Maintenance determines if the application is taken down for maintenance or not.
 	Maintenance *bool `json:"maintenance,omitempty"`
 	// Routable determines if the application should be exposed by the router.
-	Routable *bool `json:"routable,omitempty"`
+	Routable  *bool    `json:"routable,omitempty"`
+	Whitelist []string `json:"whitelist,omitempty"`
```

Comment:
it's not actually readonly, if someone wants to use just the sdk and create a new appsettings object and send it to controller, he can still do it and the new whitelist is used by the router than the old whitelist.